### PR TITLE
add some more vanity addresses, test correctness

### PR DESCRIPTION
**Pull request by NiloCK**
these characters can be owners on some different deploy targets...

### CODE REVIEW

File: test/SCBridgeWallet.test.ts [hunk 1]
```diff
@@ -28,6 +28,56 @@ async function getBlockTimestamp(): Promise<number> {
...
+      },
+      {
+        name: "Irene",
+        address: "0x111A00868581f73AB42FEEF67D235Ca09ca1E8db",
+        key: "0xfebb3b74b0b52d0976f6571d555f4ac8b91c308dfa25c7b58d1e6a7c3f50c781",
+      },
+    ];
+
+    characters.forEach((c) => {
+      // const signingKey = new ethers.SigningKey(getBytes(c.key));
```
Comment (bitwiseguy) [inline test/SCBridgeWallet.test.ts:74]: Remove comment?

File: test/SCBridgeWallet.test.ts [hunk 2]
```diff
@@ -28,6 +28,56 @@ async function getBlockTimestamp(): Promise<number> {
...
+
+      {
+        name: "Fred",
+        address: "0xfff1b9434a6B6402508911028545Aa9CE080ab12",
+        key: "0x1234f96c85acb25dff1f20e03ba287a008c99a02541ba5a9ae8b4392dd12d4ff",
+      },
+      {
+        name: "Irene",
+        address: "0x111A00868581f73AB42FEEF67D235Ca09ca1E8db",
+        key: "0xfebb3b74b0b52d0976f6571d555f4ac8b91c308dfa25c7b58d1e6a7c3f50c781",
```
Comment (bitwiseguy) [inline test/SCBridgeWallet.test.ts:69]: `key` is a bit ambiguous. Suggest using either `privateKey` or `secretKey` instead

File: test/SCBridgeWallet.test.ts (hunk 1 above)
Comment (NiloCK) [reply test/SCBridgeWallet.test.ts]: [15fc7c9](https://github.com/statechannels/SCBridge-Wallet/pull/160/commits/15fc7c9ebd2c29397a623bf0dbc32ac609b8476b)

File: test/SCBridgeWallet.test.ts (hunk 2 above)
Comment (NiloCK) [reply test/SCBridgeWallet.test.ts]: [15fc7c9](https://github.com/statechannels/SCBridge-Wallet/pull/160/commits/15fc7c9ebd2c29397a623bf0dbc32ac609b8476b)

**Pull request merged**